book/src/keymap.md

```diff
@@ -32,8 +32,8 @@
 | Key                   | Description                                        | Command                     |
 | -----                 | -----------                                        | -------                     |
 | `h`, `Left`           | Move left                                          | `move_char_left`            |
-| `j`, `Down`           | Move down                                          | `move_line_down`            |
-| `k`, `Up`             | Move up                                            | `move_line_up`              |
+| `j`, `Down`           | Move down                                          | `move_visual_line_down`            |
+| `k`, `Up`             | Move up                                            | `move_visual_line_up`              |
```

Comment:
```suggestion
| `j`, `Down`           | Move down                                          | `move_visual_line_down`     |
| `k`, `Up`             | Move up                                            | `move_visual_line_up`       |
```

Reply:
Oh yeah of course, not sure why I didn't see that haha